test/integration/start_stop_delete_test.go

```diff
@@ -153,6 +153,14 @@ func TestStartStop(t *testing.T) {
 					if err != nil {
 						t.Errorf("%s failed: %v", rr.Args, err)
 					}
+
+					rr, err = Run(t, exec.CommandContext(ctx, "kubectl", "config", "get-contexts"))
+					if err != nil {
+						t.Fatalf("%s failed: %v", rr.Args, err)
+					}
+					if notDeleted := strings.Contains(rr.Output(), profile); notDeleted {
+						t.Errorf(" kubeconfig of %s is not deleted = %t; want = %t", profile, notDeleted, false)
```

Comment:
Remove the leading space, and add `rr.Output()` to the error for debugging.

Reply:
I couldn't agree with you more.
Yeah, for debugging when the error occured, rr.Output() will have much more infomation.
Thank you for reviewing @tstromberg , I will create another pr to refactoring this test after this weekend.

Have a nice weekend :)

Reply:
DONE in #6567